README.md

```diff
@@ -72,27 +81,27 @@ yay -S ani-cli
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/ani-cli.svg)](https://repology.org/project/ani-cli/versions)
 
-### Linux
+### Linux and Mac OS
 
 Install dependencies [(See below)](#Dependencies)
 
 ```sh
-sudo curl -sL github.com/pystardust/ani-cli/raw/master/ani-cli -o /usr/local/bin/ani-cli &&
-sudo chmod +x /usr/local/bin/ani-cli
+git clone https://github.com/pystardust/ani-cli.git && cd ani-cli
+chmod +x ani-cli && sudo cp ani-cli /usr/local/bin/ani-cli
+chmod +x UI && sudo cp UI /usr/local/bin/UI
+sudo cp -r players /usr/local/bin/players
```

Comment:
apple silicon macs use a different installation path for brew than intel ones
and if the user already has brew then $(brew --prefix) is already safe and is in path

Reply:
<img width="1233" alt="Screen Shot 2022-06-15 at 1 16 54 PM" src="https://user-images.githubusercontent.com/73036332/173772076-c853b5dd-853d-4942-aaba-293cd911812b.png">